### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: bin/node/cli/src/chain_spec.rs [hunk 1]
```diff
@@ -64,7 +64,7 @@ pub fn pangolin_session_keys(
 pub fn properties() -> Properties {
 	let mut properties = Properties::new();
 
-	properties.insert("ss58Format".into(), 42.into());
+	properties.insert("ss58Format".into(), 18.into());
```
Comment (HackFisher) [inline bin/node/cli/src/chain_spec.rs:67]: Are we going to change Pangolin ss format to 18?

**Review by HackFisher — CHANGES_REQUESTED** [review]

File: bin/node/runtime/pangolin/src/lib.rs [hunk 2]
```diff
@@ -74,18 +74,18 @@ pub mod impls {
 			fn affirm_time(round: u32) -> Self::Moment {
 				match round {
 					// 1.5 mins
-					0 => 15,
+					0 => 30,
```
Comment (AsceticBear) [inline bin/node/runtime/pangolin/src/lib.rs:77]: The comment should update also?

Comment (AurevoirXavier) [reply bin/node/runtime/pangolin/src/lib.rs:77]: Block time halved.

File: bin/node/runtime/pangolin/src/lib.rs [hunk 3]
```diff
@@ -303,19 +390,15 @@ type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
...
-	<Runtime as frame_system::Config>::AccountId,
->>::NegativeImbalance;
-
 /// This runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("Pangolin"),
 	impl_name: create_runtime_str!("Pangolin"),
 	authoring_version: 1,
-	spec_version: 13,
+	spec_version: 20,
```
Comment (AsceticBear) [inline bin/node/runtime/pangolin/src/lib.rs:398]: Update spec_version from 13 to 20 directly, why not 14?

File: bin/node/cli/src/chain_spec.rs (hunk 1 above)
Comment (AurevoirXavier) [reply bin/node/cli/src/chain_spec.rs:67]: Yes, already live in pangolin-rpc.darwinia.network

@AsceticBear @WoeOm want to make keep same with mainnet which good for testing.

File: bin/node/runtime/pangolin/src/lib.rs (hunk 3 above)
Comment (AurevoirXavier) [reply bin/node/runtime/pangolin/src/lib.rs:398]: This's a re-genesis. Not runtime upgrade.

**Comment by AsceticBear** [comment]
The DVM part looks good to me.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**